### ISSUE TITLE
Validation fails for jsonschema version 4.18

### ISSUE THREAD

**Issue by shashigharti** [frictionlessdata/frictionless-py#1569]
# Overview

This fails in jsonschema version [4.18](url) 

```
schema = {
    "$schema": "http://json-schema.org/draft-04/schema#",
    "title": "Camera Trap Data Package Profile",
    "type": "object",
    "allOf": [
        {
         "$ref": "https://frictionlessdata.io/schemas/data-package.json"
        },
        {
          "required": [
            "multimedia_access",
            "project",
            "spatial",
            "temporal",
            "taxonomic"
          ]
        }
    ]  
}
descriptor = {'profile': 'camtrap.json', 'resources': [{'name': 'table', 'type': 'table', 'path': 'table.csv', 'scheme': 'file', 'format': 'csv', 'mediatype': 'text/csv'}]}
validator_class = jsonschema.validators.validator_for(schema)
validator = validator_class(schema)
for error in validator.iter_errors(descriptor):
    print(error)

# jsonschema.exceptions._WrappedReferencingError: Unresolvable: https://frictionlessdata.io/schemas/data-package.json

```
Due to that the following code fails:

```
from frictionless import Package, Resource, FrictionlessException
import pytest

profile = "camtrap.json"
resource = Resource(name="table", path="table.csv")
package = Package(resources=[resource], profile=profile)
with pytest.raises(FrictionlessException) as excinfo:
    package.to_descriptor(validate=True)
```

**Comment by pschumm** [comment]
I would just point out that [jupyter_events](https://github.com/jupyter/jupyter_events) has now [moved to jsonschema v4.18.0](https://github.com/jupyter/jupyter_events/pull/80), so this creates a problem for those using Frictionless within JupyterLab (for the moment, one can get around this by pinning jupyter_events<v0.7.0). My sense (based on a quick read) is that this is going to require some code changes to frictionless-py to make it compatible with the new API implemented in jsonschema>=4.18.0. Unfortunately I don't have time to work on this right now.

**Comment by johentsch** [comment]
> so this creates a problem for those using Frictionless within JupyterLab

Yes I second that. Frictionless is a core dependency to our library, which is also designed for working well in Jupyter, so there is a dependency clash

> frictionless 5.16.0 requires jsonschema<4.18

**Comment by johentsch** [comment]
Any prospects of this being addressed in the near future?

**Comment by zaneselvans** [comment]
I finally updated our project to use frictionless v5 rather than v4 and was also surprised to see `jsonschema` get downgraded to a version that's almost a year and a half old with the attendant downgrades of multiple Jupyter packages.

It would be great to get this updated!

**Comment by pschumm** [comment]
FYI, the latest JupyterLab releases (4.1.8 and 4.2.0rc0) now require `jupyterlab_server` 2.27.1, which itself requires `jsonschema` 4.18.0. So this is the end of the road for JupyterLab updates with Frictionless until this issue is addressed.

**Comment by pschumm** [comment]
I believe this issue can now be closed, since following 925cbb3 the `jsonschema` dependency is now

    jsonschema>=4.20

@roll, do you agree?

**Comment by roll** [comment]
Thanks, yes!
